lib/bounces.js

```diff
@@ -120,14 +120,7 @@ module.exports = function (log, error) {
 
           // Flow events have a specific event and structure that must be emitted.
           // Ref `gather` in https://github.com/mozilla/fxa-auth-server/blob/master/lib/metrics/context.js
-          const flowEventInfo = {
-            event: eventName,
-            time: currentTime,
-            flow_id: flowId,
-            flow_time: elapsedTime
-          }
-
-          log.flowEvent(flowEventInfo)
```

Comment:
If `log.flowEvent` isn't working for some reason, we should try to fix it so that it works properly rather than branch off and do something different here as a special case.

At the moment I don't quite understand what the problem is from your description in https://github.com/mozilla/fxa-auth-server/issues/1425#issuecomment-273301265. Maybe we should have a chat over vidyo when you get online?